.github/workflows/merge-release-branch.yml

```diff
@@ -0,0 +1,22 @@
+name: Merge Release Into Master
+
+on: repository_dispatch
+
+jobs:
+  merge_to_master:
+    runs-on: ubuntu-latest
+    # Allow GITHUB_TOKEN to have write permissions
+    permissions:
+      contents: write
```

Comment:
Is the goal here to grant `GITHUB_TOKEN` with default permissions (without write access) and add write access in the workflow?